package.json

```diff
@@ -32,35 +32,35 @@
     "babel-eslint": "^10.0.2",
     "babel-jest": "^24.8.0",
     "babel-loader": "^8.0.2",
-    "copy-webpack-plugin": "^4.5.1",
-    "core-js": "^3.1.4",
-    "crowdin-api": "^2.0.2",
-    "css-loader": "^1.0.0",
+    "copy-webpack-plugin": "^5.0.0",
+    "core-js": "3",
```

Comment:
This was already pegged at `^3.1.4`, could it be at least that or higher?

Reply:
this also installs the latest from v3 but i pegged it at that version now